wiki/docs/en/learn-docker/basic-knowledge.md

```diff
@@ -0,0 +1,76 @@
+
+- **Container Creation**: When a Docker image is run, it creates a Docker container. The container is a virtual environment that encapsulates the application code along with all its dependencies, ensuring the application runs quickly and reliably across different computing environments.
+
+## What is a Docker Container?
+
+A Docker container is a runtime instance of a Docker image. It allows developers to package applications along with all the necessary dependencies, such as libraries and configurations, into a single unit.
+
+- **Runtime Instance**: A container is essentially a live instance of an image that is running as a process on the host machine. It encapsulates the application code and its runtime environment, ensuring the application can run in an isolated and consistent manner across different environments.
+
+- **Isolated Execution**: Containers provide isolation for applications, meaning they run independently of the host and other containers. This isolation ensures that changes or dependencies within one container do not affect others, enhancing security and reliability.
+
+- **Example**: For instance, if you have a Docker image containing Ubuntu OS with an Nginx server configured, running this image with the `docker run` command will create a container. This container will then run the Nginx server within the Ubuntu OS environment, isolated from other containers and the host system.
+
+
+## What is Docker Hub?
+
+Docker Hub is a cloud-based repository service where developers and DevOps teams can store, manage, and share Docker container images. It serves as a centralized resource for Docker images, allowing users to push their own images for private or public use and pull images created by others.
+
+- **Repository Service**: Docker Hub acts as a repository where Docker images are stored and organized. Users can upload their Docker images to Docker Hub for storage and distribution.
+
+- **Cloud-Based Service**: It operates on the cloud, making it accessible from anywhere via the internet. This accessibility allows developers to easily find and reuse Docker images across different projects and environments.
+
+- **Public and Private Registries**: Docker Hub supports both public and private registries. Public registries allow images to be shared openly with the community, while private registries restrict access to authorized users or teams.
+
+- **Usage in DevOps**: Docker Hub is widely used by DevOps teams for managing containerized applications. It facilitates collaboration by enabling teams to share and deploy standardized images reliably.
+
+- **Availability and Requirements**: Docker Hub is freely available for all operating systems and is an essential tool for anyone working with Docker containers. Users accessing Docker Hub need basic knowledge of Docker commands to push and pull images effectively.
+
```

Comment:
We could keep it vendor neutral and mention Docker Hub as one of the available solutions for a **Container registry**. This section Could be titled  **What is a Container Registry** and keep your explanations which still apply to it.